solutions/community-deploy/CORTX-Deployment.md

```diff
@@ -56,37 +56,37 @@ git clone https://github.com/Seagate/cortx-re && cd $PWD/cortx-re/solutions/kube
 ```
 - Create the hosts file in the current directory to add entries for all the nodes with same format in hosts file.
 
-**For Example:** 
-`hosts` file for multi-node setup is as below,
 ```
 hostname=cortx-deploy-node1.cortx.com,user=root,pass=<root-password>
 hostname=cortx-deploy-node2.cortx.com,user=root,pass=<root-password>
 hostname=cortx-deploy-node3.cortx.com,user=root,pass=<root-password>
+hostname=cortx-deploy-node4.cortx.com,user=root,pass=<root-password>
+```
+- Following command will modify the `hosts` file for all the nodes on your AWS EC2 instances
+**Note:** You can provide your password for the root user
+```
+export ROOT_PASSWORD=<YOUR_ROOT_PASSWORD>
+for ip in $PUBLIC_IP;do ssh -i cortx.pem -o 'StrictHostKeyChecking=no' centos@"${ip}" "git clone https://github.com/Seagate/cortx-re && pushd /home/centos/cortx-re/solutions/kubernetes && touch hosts && echo hostname=${HOST1},user=root,pass= > hosts && sed -i 's,pass=.*,pass=$ROOT_PASSWORD,g' hosts && echo hostname=${HOST2},user=root,pass= >> hosts && sed -i 's,pass=.*,pass=$ROOT_PASSWORD,g' hosts && echo hostname=${HOST3},user=root,pass= >> hosts && sed -i 's,pass=.*,pass=$ROOT_PASSWORD,g' hosts && echo hostname=${HOST4},user=root,pass= >> hosts && sed -i 's,pass=.*,pass=$ROOT_PASSWORD,g' hosts && cat hosts";done
 ```
-- To execute `cluster-setup.sh` to setup K8s cluster on your AWS EC2 instances for multi-node deployment follow,
-   - copy `/tmp/daemon.json` file from primary node to all the worker nodes in `/etc/docker/daemon.json` path either by `rsync` command or vi editor or,
-   - copy `/etc/docker/daemon.json` file content from primary node to worker nodes on same target path to update the local registry node.
-   - Then restart docker service on all the worker nodes by running, `systemctl restart docker`
 - To allow the PODs creation on primary node, pass the first input parameter for cluster-setup.sh script as true. Please note you must pass the input parameter as true for multi-node setup.
 ```
 ./cluster-setup.sh true
 ```
+- To execute `cluster-setup.sh` to setup K8s cluster on your AWS EC2 instances which modify `/etc/docker/daemon.json` on all the nodes and then restart docker deamon.
```

Comment:
Please move AWS specific instructions to https://github.com/Seagate/cortx-re/blob/main/solutions/community-deploy/cloud/AWS/README.md

Reply:
done